### PR TITLE
Added Solution of Gas Station problem can 

### PR DESCRIPTION
**Pull request by deepesh85b**
@ignacio-chiazzo 
#49 
Please review and merge this.
I have added Solution of this(https://leetcode.com/problems/gas-station/description/)

### CODE REVIEW

**Review by ignacio-chiazzo — COMMENTED** [review]
Nice solution! 

Could you add an entry in the [README](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript#readme) file?

File: LeetcodeProblems/Algorithms/GasStation/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/**
+ * Problem: https://leetcode.com/problems/gas-station/description/
```
[the remark this answers is not in the document]
Comment (deepesh85b) [reply LeetcodeProblems/Algorithms/GasStation/index.js:2]: > Could you copy the problem's description here please?
You can merge it now
@ignacio-chiazzo 
#49

File: LeetcodeProblems/Algorithms/GasStation/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  for (var i = 0; i < len; ++i) {
+    gasSum += gas[i];
+    costSum += cost[i];
+  }
+
+  if (costSum > gasSum)
+    return -1;
+  else {
+    gasSum = costSum = 0;
+    for (var i = 0; i < len; ++i) {
```
Comment (ignacio-chiazzo) [inline LeetcodeProblems/Algorithms/GasStation/index.js:37]:
```suggestion
    for (i = 0; i < len; ++i) {
```

**Review by ignacio-chiazzo — COMMENTED** [review]
It looks like the linter failed.https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/actions/runs/3265186656/jobs/5373392711

**Pull request merged**